planex/cmd/override.py

```diff
@@ -0,0 +1,133 @@
+"""
+planex-override: generate override files pointing at local repos
+in xenserver-specs/repos/
+"""
+
+import errno
+import json
+import os
+
+import argparse
+import argcomplete
+
+from planex.git import current_branch
+from planex.util import add_common_parser_options
+
+
+def mkdir_p(path):
+    """
+    Like `mkdir -p path` but does not fail when the path is already present.
+    """
+    try:
+        os.makedirs(path)
```

Comment:
You explained another good reason: in a concurrent environment, there is a chance to create the directory between `isdir(path)` and `makedirs(path)`.